command-snapshot.json

```diff
@@ -2,13 +2,37 @@
   {
     "command": "force:limits:api:display",
     "plugin": "@salesforce/plugin-limits",
-    "flags": ["apiversion", "json", "loglevel", "targetusername"],
-    "alias": []
+    "flags": ["json", "target-org"],
+    "alias": ["force:limits:api:display", "show:api:limits"]
   },
   {
     "command": "force:limits:recordcounts:display",
     "plugin": "@salesforce/plugin-limits",
-    "flags": ["apiversion", "json", "loglevel", "sobjecttype", "targetusername"],
-    "alias": []
+    "flags": ["json", "sobject-type", "target-org"],
```

Comment:
over in `data` Matt and I decided to go with `sobject` instead of `sobject-type`.

Reply:
I will change the sf flag to `sobject` and leave the alias as is